Automattic/jetpack
# Publicize: Add REST API

Replaces #10372. Based on #10396. ~Rebase on `master` after merging #10396!~ (done)

#### Changes proposed in this Pull Request:

Add a REST API for Publicize.

#### Testing instructions:

See companion PR https://github.com/Automattic/wp-calypso/pull/28102.

This doesn't yet fully replicate the non-Gutenberg UI (e.g. no warning message next to a Facebook connection; publicize message placeholder; expansion behavior). I'd like to tackle those in a subsequent iteration.

#### Proposed changelog entry for your changes:

* Add a REST API for Publicize.
